src/main/java/org/jboss/arquillian/testcontainers/TestContainerInstances.java

```diff
@@ -0,0 +1,46 @@
+package org.jboss.arquillian.testcontainers;
+
+import java.util.List;
+
+import org.testcontainers.containers.GenericContainer;
+
+public class TestContainerInstances {
+    private List<GenericContainer<?>> containers;
+
+    public TestContainerInstances(List<GenericContainer<?>> containers) {
+        this.containers = containers;
+    }
+
+    public List<GenericContainer<?>> all() {
+        return containers;
+    }
+
+    public GenericContainer<?> get(int index) {
+        return containers.get(index);
+    }
```

Comment:
Should we include a `get(Class<? extends Annotation> qualifier)` to lookup by the qualifier?

Reply:
Done